core/java/src/org/openda/blackbox/wrapper/BBModelInstance.java

```diff
@@ -327,14 +327,14 @@ private void addAllExchangeItemIdsFromVectorConfig(BBModelVectorConfig vectorCon
 	}
 
 	public IExchangeItem getDataObjectExchangeItem(String exchangeItemID) {
-		IPrevExchangeItem exchangeItem = getExchangeItem(exchangeItemID);
+		IExchangeItem exchangeItem = getExchangeItem(exchangeItemID);
 		if (!(exchangeItem instanceof BBExchangeItem)) {
 			throw new RuntimeException("Unexpected exchange item type " + exchangeItem.getClass().toString());
 		}
 		return (BBExchangeItem) exchangeItem;
 	}
 
-	public IPrevExchangeItem getExchangeItem(String searchedBBExchangeItemId) {
+	public IExchangeItem getExchangeItem(String searchedBBExchangeItemId) {
```

Comment:
Codacy found an issue: [The method 'getExchangeItem(String)' has an NPath complexity of 213](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)